bluebrain/repo-bluebrain/packages/py-clang-format/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+from spack import *
```

Comment:
Should be in Patches unless its BBP code

Reply:
Done in 495801350d1fe4359e898ed34ed88ae151244b11